.gitpod/Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM gitpod/workspace-full
-
+FROM gitpod/workspace-full:latest
+ARG JAVA_VERSION=21.0.6-tem
 ARG MAVEN_VERSION=3.9.9
-
-RUN brew install gh && \
-    bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && sdk install maven ${MAVEN_VERSION} && sdk default maven ${MAVEN_VERSION}"
+# Install Java 21, Maven and GitHub CLI
+RUN bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && \
+    sdk install java ${JAVA_VERSION} && \
+    sdk default java ${JAVA_VERSION}" && \
+    bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && \
```

Comment:
why are you running bash -c again and loading sdkman again? I don't think this is needed.